cnxepub/tests/test_formatters.py

```diff
@@ -433,6 +433,10 @@ def test_binder(self):
             expected_content = f.read()
 
         actual = str(SingleHTMLFormatter(self.desserts))
+        # Uncomment this if/when you get a test failure
```

Comment:
Hm... Not sure if it's a good idea.  It's not that hard to add this if necessary is it?